subprojects/docs/src/docs/release/notes.md

```diff
@@ -53,6 +53,14 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
+### Change of default Checkstyle configuration directory
+
+Configuring rules consumed by the Checkstyle plugin in a multi-project build used to be tedious.
+Every sub project needs to provide a configuration file even if the content is exactly the same.
+With this release of Gradle, the Checkstyle configuration file only needs to be provided in the directory `config/checkstyle` of the root project and automatically applies to all sub projects.
+Setting the Checkstyle configuration file per sub project has been deprecated and is scheduled to be removed with the next major Gradle version.
```

Comment:
This is misleading. We didn't deprecate setting it per subproject. This should still be valid if the user explicitly does so.

Reply:
Removed the sentence. However, the deprecation message indicates that if I'd read the message as a user. Sure, you can directly point to it but nevertheless the message is "don't do that". Let me know if you think that the message should be changed as well.

Reply:
The message shouldn't be shown when the user sets it explicitly.

Reply:
Added a test case that demonstrates the functionality.